bricks/forms/src/cmdb-instance-select/CmdbInstanceSelect.tsx

```diff
@@ -156,7 +158,7 @@ export function CmdbInstanceSelectItem(
               ...extraQuery,
             ],
           },
-
+          ...(permission ? { permission } : {}),
```

Comment:
```suggestion
          permission,
```

这样就好了吧？undefined 在调用接口时会被忽略。